qml/components-generic/VenueDetails.qml

```diff
@@ -26,6 +26,7 @@ import QtQuick 2.2
 import Sailfish.Silica 1.0
 
 import BerlinVegan.components.generic 1.0 as BVApp
+import BerlinVegan.components.platform 1.0 as BVPlatform
```

Comment:
You can import both the generic as well as the platform specific components into the same `BVApp` namespace.